### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by Nainterceptor**
Checklist

- [x] I've added a mention in CHANGELOG.md

### CODE REVIEW

**Comment by Scttpr** [comment]
Should `static/background.jpg` be pushed ?

File: components/planning/cars/calendar.vue [hunk 1]
```diff
@@ -19,7 +19,7 @@
         @click-and-release="openCreate"
         @view-change="viewChange"
       >
-        <template #event-renderer="{ event: { content }, view }">
```
Comment (Scttpr) [inline components/planning/cars/calendar.vue]: Not sure but view is not furnished anymore ?

File: components/planning/drivers/calendar.vue [hunk 2]
```diff
@@ -19,7 +19,7 @@
         @click-and-release="openCreate"
         @view-change="viewChange"
       >
-        <template #event-renderer="{ event: { content }, view }">
```
Comment (Scttpr) [inline components/planning/drivers/calendar.vue]: Same here

**Comment by Nainterceptor** [comment]
[Ajouter header pour empecher la création d'une iframe sur un site tier](https://trello.com/c/vwISM7UP/289-ajouter-header-pour-empecher-la-cr%C3%A9ation-dune-iframe-sur-un-site-tier)

File: components/planning/cars/calendar.vue (hunk 1 above)
Comment (Nainterceptor) [reply components/planning/cars/calendar.vue]: view is not used in the template block

File: components/planning/drivers/calendar.vue (hunk 2 above)
Comment (Nainterceptor) [reply components/planning/drivers/calendar.vue]: view is not used in the template block

**Comment by Nainterceptor** [comment]
> Should `static/background.jpg` be pushed ?

This file is not longer used since you've changed login page

**Pull request merged**